### PR TITLE
home page shortcut link to counselor toolkit

### PR DESCRIPTION
**Pull request by acedeywin**
Primary reviewer: @stephenhand 

## Description

- New feature: Home page shortcut link to counselor toolkit

### Checklist
- [x] Corresponding issue has been opened
- [N/A] New tests added
- [x] Feature flags added
- [x] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes CHI-1547

### Verification steps

- Verify that there is an information icon on the side nav
- Click on the information Icon on the side nav
Screenshot 2023-02-02 at 11 20 05 AM

### CODE REVIEW

File: plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import React from 'react';
+
+import CounselorToolkitSideLink from './CounselorToolkitSideLink';
+import { getConfig } from '../../HrmFormPlugin';
+
+// eslint-disable-next-line import/no-unused-modules
+export const setUpCounselorToolkits = () => {
+  const { featureFlags } = getConfig();
+  // eslint-disable-next-line no-unused-expressions
+  featureFlags.enable_counselor_toolkits &&
```
Comment (stephenhand) [inline plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx:27]: We should also check if a kmsUrl is configured on the default helpline before showing this icon, in addition to the feature flag

File: plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import * as Flex from '@twilio/flex-ui';
+import React from 'react';
+
+import CounselorToolkitSideLink from './CounselorToolkitSideLink';
+import { getConfig } from '../../HrmFormPlugin';
```
Comment (stephenhand) [inline plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx:21]: `getConfig` from `HrmFormPlugin` is deprecated, use `getHrmConfig` from `hrmConfig` instead

File: plugin-hrm-form/src/styles/global-overrides.css [hunk 3]
```diff
@@ -43,6 +43,10 @@ div.Twilio-MainHeader button.MuiButton-root {
     background-color: rgb(18, 28, 45);
     border: none;
 }
+div.css-1nnjdk7:hover {
```
Comment (stephenhand) [inline plugin-hrm-form/src/styles/global-overrides.css:46]: I'm not sure what effect this is implementing but we can't override the autogenerated CSS class names like this - as well as being very obfuscated and hard to maintain, these generated names could change at any time, andf the effect would be lost

We need to try to achieve this with CSS that doesn't reference one of these class names, or if this is not achievable we need to go back to Jana & Dee

**Review by stephenhand — CHANGES_REQUESTED** [review]
Couple of changes, the only tricky one might be the CSS override

File: plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx (hunk 2 above)
Comment (acedeywin) [reply plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx:21]: @stephenhand `featureFlags` does not exist in `getHrmConfig` but in `getConfig` of `hrmConfig`.

Comment (stephenhand) [reply plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx]: `getAseloFeatureFlags()` can be used to get the feature flages from `hrmConfig`

Comment (acedeywin) [reply plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx]: @stephenhand Done!

File: plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx (hunk 1 above)
Comment (stephenhand) [reply plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx:27]: @acedeywin - this one is still outstanding

**Review by stephenhand — APPROVED** [review]

**Review by stephenhand — CHANGES_REQUESTED** [review]
Oops, forgot there was 1 change still outstanding

File: plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx (hunk 1 above)
Comment (acedeywin) [reply plugin-hrm-form/src/components/toolkits/setUpCounselorToolkits.tsx:27]: @stephenhand, I have implemented the changes. I did that in the `plugin-hrm-form/src/components/toolkits/CounselorToolkitSideLink.tsx` file.

**Review by stephenhand — APPROVED** [review]

**Pull request merged**